Tests/SwiftPlotTests/Annotation/annotation-text.swift

```diff
@@ -30,11 +30,13 @@ extension AnnotationTests {
                                      renderer: agg_renderer)
     verifyImage(name: fileName, renderer: .agg)
     #endif
+    /*
```

Comment:
@WilliamHYZhang you need not comment out this part. The #if canImport will take care of it if you're not on macOS.

Reply:
The tests seem to fail if this line is commented out, is that ok?

Reply:
This test shouldn't fail if it isn't commented out. I tried it. Because of the #if canImport this code shouldn't compile.

Reply:
Oh, I was reffering to the PE build test. When I had that line uncommented and pushed a commit, I kept on getting an email that the PR test failed.

Reply:
I'm not sure but that could have been due to the image you had added to the references folder. Could you try uncommenting it?

Reply:
I think the error was that the file was not found, if there was no file in the quartz yet the tester still tested it.

Reply:
Oh alright I understand now. Github actions builds it on macOS. You're right. 
Could you just remove the PlotAnnotation struct, update the example and then someone else can add the reference image for quartz.

Reply:
The issue is that the test server does compile the Quartz part and goes and checks if the file is there and it isn’t so it fails. Long term we should come up with a solution for this.

Reply:
The issue is that the test server does compile the Quartz part and goes and checks if the file is there and it isn’t so it fails. Long term we should come up with a solution for this.

Reply:
I don't understand what happened with the GitHub comments, I commented once, two identical comments were posted for some reason, after refreshing the page they were still there so I deleted one and then the comment of @KarthikRIyer appeared before mine... some caching issue I guess.

Reply:
Alright, will do!